README.md

```diff
@@ -117,3 +130,53 @@
 - [ ] HA of databases and other persistent services.
 
 ## Advance MLonCode awareness
+
+### Make source{d} the face of #MLonCode
+- [ ] Run that workshop at least once at a meetup / conference
+- [ ] Create two demos showing the power of our stack solving a “real problem”
+- [ ] Get at least 6 more collaborators (talk/blog/...) with engineering
+- [ ] Improve the formal process of our internal speaker program in collaboration with Esther and get buyin from Engineering
+
+### Create an "MLonCode friends" program
+- [ ] Define "MLonCode friends" program goals & value proposition
+- [ ] Identify potential members and invite them officially, get 20 to join
+- [ ] Welcome each member personally and provide some swag
+- [ ] Classify members by profile (speakers, researchers, industry/academia, event organizers, etc)
+- [ ] Engage with members and enable them to grow their influence and our message
+
+### Make source{d} a friend of MLonCode academia  
+- [ ] Create a list of all academic events relevant to MLonCode until end of 2019
+  - [ ] Decide level of involvement (attending, presenting, sponsoring, etc)
+- [ ] Release CodeNet with one challenge on top of PGA
+- [ ] Consider and plan/reject creation of competition on Kaggle
+- [ ] 3 citations of source{d} datasets / tech in academia endorsed documents
+- [ ] Develop and organize at least 3 Enterprise workshops in Q3
+
+### Promote current source{d} stack
+- [ ] Write and promote 1 blog post for each project in the stack and how it came to be
+- [ ] Gather list of users of each project and classify them according to their profiles
+- [ ] source{d} Engine and/or Automated Code Review Bot featured at minimum 5 conferences
+- [ ] source{d} Engine and/or Automated Code Review Bot featured at minimum 5 meetups
+
+### Define PR & BD Strategy
+- [ ] Develop & pitch source{d}'s unique Open Source Philosophy / Culture to the press and get publication in at least 2 publications
+- [ ] Onboard a PR agency to support product annoucements
+- [ ] Identify at least 1 BD led opportunity for joint announcement i.e GitHub / GitLab integration, etc
+
+### Define Social Media Strategy
+- [ ] Define infrastructure for this (Buffer, etc)
+- [ ] Slack: figure out strategy and onboarding process
+- [ ] Increase followers on Twitter / Facebook /  Linkedin ...
+- [ ] Provide an alternate method to Slack for communication: forum?
+
+### Create a weekly newsletter for announcements, events, curated content, etc
+- [ ] Create the newsletter
+- [ ] Curate content and send newsletter weekly
+- [ ] Reach 100 subscribers
+
+### Increase social engagement with #MLonCode #49
+- [ ] Increase hashtag usage on social media channels by X% (TBD with #48)
```

Comment:
@vcoisne can you clarify the meaning of `#48` here

Reply:
He meant #48, specifically point one: `[P0] Describe the metrics of devrel success`